Examples/Sources/MacroExamples/Implementation/Expression/FontLiteralMacro.swift

```diff
@@ -35,14 +35,14 @@ private func replaceFirstLabel(
   of tuple: LabeledExprListSyntax,
   with newLabel: String
 ) -> LabeledExprListSyntax {
-  guard let firstElement = tuple.first else {
+  guard var firstElement = tuple.first else {
     return tuple
   }
 
+  firstElement.label = .identifier(newLabel)
+  firstElement.colon = .colonToken()
+
   var tuple = tuple
-  tuple[tuple.startIndex] =
-    firstElement
-    .with(\.label, .identifier(newLabel))
-    .with(\.colon, .colonToken())
+  tuple[tuple.startIndex] = firstElement
   return tuple
```

Comment:
Can we simplify this further to

```swift
guard !tuple.isEmpty else {
  return tuple
}
tuple[tuple.startIndex].label = .identifier(newLabel)
tuple[tuple.startIndex].colon = .colonToken()
return tuple
```

Reply:
Great idea! One suggestion: what are your thoughts on using `if` instead of `guard` for future considerations as well? I find that avoiding negation makes the code easier to read, rather than using it solely to justify a `guard` statement.

```swift
  if tuple.isEmpty {
    return tuple
  }

  var tuple = tuple
  tuple[tuple.startIndex].label = .identifier(newLabel)
  tuple[tuple.startIndex].colon = .colonToken()
  return tuple
```

Reply:
I don’t have strong opinions here. Pick whichever you like better.

Reply:
I feel you. I've picked `if`.